### PR TITLE
google and aws api implementation

### PR DESCRIPTION
**Pull request by PratikDhanave**
google serverless api
google bigtable api
aws dynamodb api
aws machine learning api
test added
documentation added

### CODE REVIEW

**Comment by PratikDhanave** [comment]
updated conflict resolved

File: ali/ali.go [hunk 1]
```diff
@@ -15,4 +17,6 @@ type Ali struct {
 	aliloadbalancer.Aliloadbalancer
 	alicontainer.Alicontainer
 	alidns.Alidns
-}
\ No newline at end of file
+	aliserverless.Aliserverless
```
Comment (OddCN) [inline ali/ali.go]: thank you for this

Comment (PratikDhanave) [reply ali/ali.go]: +1

**Comment by PratikDhanave** [comment]
conflicts resolved other pull request created

**Pull request closed without merging**